utils/test_reporter.go

```diff
@@ -86,3 +86,7 @@ func (r *TestReporter) Errorf(format string, args ...interface{}) {
 	r.T.Helper()
 	r.T.Errorf(format, args...)
 }
+
+func (r *TestReporter) Cleanup(cb func()) {
+	r.T.Helper()
+}
```

Comment:
no call?

Reply:
> I'll add it.
I don't say we need it, just asking how we add a function that we don't use. what is the reason for this one

Reply:
It's required because it must implement the interface. This is a new function added to the interface.